File: palindrome.cpp
#include<iostream>
using namespace std;
int main () {
    int num;
    cout << "Enter the num: ";
    cin >> num;

    int original = num;
    int rev = 0;

    while (num > 0) {
        int digit = num % 10;
        rev = rev*10 + digit;
        num = num/10;
    }

    if (original == rev) {
        cout << "Given number is a Palindrome";
    }else {
        cout << "Given number is not a Plaindrome";
    }

}

File: Palindrome.js
let num = 54145;
let original = num;
let rev = 0;

while (num > 0) {
  let digit = num % 10;
  rev = rev * 10 + digit;
  num = Math.floor(num / 10);
}

if (original == rev) {
  console.log("Palindrome")
} else {
  console.log("not Plaindrome");
}
